doc/source/parameters/dem/insertion_info.rst

```diff
@@ -119,7 +123,7 @@ The ``list`` insertion method insert particles at precis coordinates with specif
 ---------------------
 File
 ---------------------
-The ``file`` insertion method insert particles in a similar way to the ``list`` insertion method. The main difference between these two methods is the option to use an external file provided by the ``insertion file name`` parameter. This parameter is set at ``particles.input`` by default. This file has to follow this structure:
+The ``file`` insertion method insert particles in a similar way to the ``list`` insertion method. The main difference between these two methods is the option to use external files provided by the ``list of input files`` parameter. This parameter is set at ``particles.input`` by default, but a list a files can be specified. At each insertion time step, a different file will be used. If the end of the list is reached and there are still particles to be inserted, the list returns to the first file. An insertion file must follow this structure:
```

Comment:
If you change the name of the parameters, don't forget to change them here :)

Reply:
Good catch